demos/try12_Deconvolution_BayesianProblem.py

```diff
@@ -26,8 +26,7 @@
 
 # %% Prior choices (Main ones of interest: Gaussian, GMR, Cauchy_diff, Laplace_diff:
 # Working choices
-#TP.prior = Gaussian(mean=np.zeros(n), std=par, geometry=TP.model.domain_geometry) #Not for 2D.
-#TP.prior = GaussianCov(mean=np.zeros(n), cov=par**2, geometry=TP.model.domain_geometry)
+#TP.prior = Gaussian(mean=np.zeros(n), cov=par**2, geometry=TP.model.domain_geometry)
```

Comment:
This is a try script with cases to comment out. Lets leave it since its not relevant to this PR. I would think when possible we need to get rid of most of these try scripts anyway and make them into tutorials/howtos/technical demos etc.